src/App.js

```diff
@@ -6,26 +6,27 @@ function App() {
   const [error, setError] = useState(null);
   const [isLoaded, setIsLoaded] = useState(false);
   const [city, setCity] = useState("New York City")
-  const [results, setResults] = useState(null);
+  const [results, setResults] = useState([]);
+  const [buttonClick, setButtonClick] = useState(false);
 
   useEffect(() => {
-    fetch("https://api.openweathermap.org/data/2.5/weather?q=" + city + "&units=metric" + "&appid=" + process.env.REACT_APP_APIKEY)
+    fetch("https://api.openweathermap.org/data/2.5/weather?q=" + city + "&units=metric&appid=" + process.env.REACT_APP_APIKEY)
```

Comment:
Code improvement suggestion: Please use template literals (a.k.a backticks) for string concatenation